spacepy/pybats/ram.py

```diff
@@ -1071,12 +1072,13 @@ def __init__(self, path='.', rotate=True, *args, **kwargs):
         self._y = np.arange(0, 26) - 0.5
         self._lt_labels = ['Dusk', 'Midnight', 'Dawn']
         self._yticks = [6.0, 12.0, 18.0]
-        self._dtime = date2num(self['time'])  # create decimal time.
+        self._dtime = self['time'][-1] - self['time'][-2]  # Time between files
         self._zlims = {'p': [0, 40], 'R': [0, 8], 'r': [0, 60], 'n': [0, 8]}
+        self._ptime = np.append(self['time'], self['time'][-1] + self._dtime)
```

Comment:
If `_dtime` means something different than it used to, and it's not used elsewhere anymore, do we still want it as a data member, or can it just be a local variable here?